rskj-core/src/main/java/org/ethereum/util/BuildInfo.java

```diff
@@ -22,36 +22,21 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Properties;
-
 public class BuildInfo {
 
-    private static final Logger logger = LoggerFactory.getLogger("general");
-
-    private static String buildHash;
-    private static String buildBranch;
+    private final String buildHash;
+    private final String buildBranch;
 
-    public static String getBuildHash() {
-        return buildHash;
+    public BuildInfo(String buildHash, String buildBranch) {
+        this.buildHash = buildHash;
+        this.buildBranch = buildBranch;
     }
 
-    static {
-        try {
-            Properties props = new Properties();
-            InputStream is = BuildInfo.class.getResourceAsStream("/build-info.properties");
-
-            props.load(is);
-
-            buildHash = props.getProperty("build.hash");
-            buildBranch = props.getProperty("build.branch");
-        } catch (IOException e) {
-            logger.error("Error reading /build-info.properties", e);
-        }
+    public String getBuildHash() {
+        return buildHash;
     }
 
-    public static void printInfo(){
+    public void printInfo(Logger logger){
```

Comment:
Could this be a private logger?

Reply:
I found this method so superfluous that I wanted to give it _some_ kind of meaningfulness. This way you can log in the context of the caller and thus avoid handling an additional Logger in your configuration